server/schema/getzonestatusbulk.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE FUNCTION GetZoneStatusBulk(
+	zonenames varchar[],
+	out zonename varchar,
+	out zonestatus varchar
+) RETURNS SETOF record AS $$
+DECLARE
+
+BEGIN
+	IF zonenames IS NOT NULL AND array_length(zonenames, 1) > 0 THEN
```

Comment:
Is this validation required?
Isn't this going to silently fail if wrong data is passed in?
Wouldn't it be better to leave out the validation and have an error when data is wrong?

Reply:
Alternatively, you could raise an exception if data is invalid.

Reply:
This validation is necessary because we want for a request that looks like this:

 `atomiadnsclient --method GetZoneStatusBulk --arg  '[]'`

to get a response like this:

`$VAR1 = [];`

We don't want to get the exception that the data is not valid. If we throw an exception for this, we will break the existing code from running.

In case you send a request like this:

`atomiadnsclient --method GetZoneStatusBulk --arg  ''`

then you will get the exception.